src/forms/FormLines.tsx

```diff
@@ -37,7 +38,15 @@ export function FormLines(props: FormLinesProps) {
           "& > *": Css.mb2.$,
         }}
       >
-        {children}
+        {Children.map(children, (child) => {
+          if (child && typeof child === "object" && "type" in child && (child.type as any).isFormHeading) {
```

Comment:
See the comment on `FormHeading.tsx` line 27, but this is the only reliable method I could get working in storybook with fast refresh.

I.e. any approach that used `child.type === FormHeading` or `child.type === createElement(FormHeading).type` would break after hot reloads and throw off the layout b/c the `isFirst` prop would never be set (b/c the hot-patched code would never match on the `child.type` check anymore).